xknx/knxip/__init__.py

```diff
@@ -28,3 +28,35 @@
 from .search_response import SearchResponse
 from .tunnelling_ack import TunnellingAck
 from .tunnelling_request import TunnellingRequest
+
+__all__ = [
+    "KNXIPBody",
+    "CEMIFrame",
+    "ConnectRequest",
+    "ConnectResponse",
+    "ConnectionStateRequest",
+    "ConnectionStateResponse",
+    "DIB",
+    "DIBDeviceInformation",
+    "DIBGeneric",
+    "DIBSuppSVCFamilies",
+    "DisconnectRequest",
+    "DisconnectResponse",
+    "ErrorCode",
+    "KNXIPHeader",
+    "HPAI",
+    "KNXIPFrame",
+    "APCICommand",
+    "CEMIFlags",
+    "CEMIMessageCode",
+    "ConnectRequestType",
+    "DIBServiceFamily",
+    "DIBTypeCode",
+    "KNXIPServiceType",
+    "KNXMedium",
+    "RoutingIndication",
+    "SearchRequest",
+    "SearchResponse",
+    "TunnellingAck",
+    "TunnellingRequest",
+]
```

Comment:
I don't like this, but it seems 
https://github.com/XKNX/xknx/blob/0defb2649644ef8fe53ab7d375875c2366be245f/setup.cfg#L51
isn't doing what I think it should do.